server/model/monitor.js

```diff
@@ -1510,7 +1510,7 @@ class Monitor extends BeanModel {
             FROM monitor_notification
             WHERE monitor_notification.monitor_id IN (?)
         `, [
-            monitorIDs,
+            monitorIDs.join(","),
```

Comment:
I just checked and this does not pose a problem as `SQLITE_MAX_VARIABLE_NUMBER=32766` and mariadb [being larger](https://stackoverflow.com/questions/4922345/how-many-bind-variables-can-i-use-in-a-sql-query-in-mysql-5)

Reply:
> This will not work. Assuming I have the ids `1` and `2` in my db and `monitorIDs=[1,2]` this would query for `"1,2"` which is a different datatype and not in the db..

Actually, in this specific case, it works for some reason. However, in the other two cases, it doesn’t work. So I’ve changed the code in all three places now.